deploy/helm-chart/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: nginx-kubernetes-gateway
 description: NGINX Kubernetes Gateway
 type: application
 version: 0.1.0
```

Comment:
should those versions be edge for both version and appVersion? (maybe it's a separate PR to fix that)

Reply:
I'll change `appVersion` (as this impacts the generated files as well), but we can't change the `version` to `edge` because it needs to be SemVer